server/src/main/java/org/apache/iotdb/db/qp/strategy/PhysicalGenerator.java

```diff
@@ -229,20 +190,21 @@ private PhysicalPlan transformQuery(QueryOperator queryOperator)
       List<Path> suffixPaths = queryOperator.getSelectOperator().getSuffixPaths();
       List<String> originAggregations = queryOperator.getSelectOperator().getAggregations();
 
-      List<String> measurementColumnList = new ArrayList<>();
-      Map<String, Set<String>> measurementColumnsGroupByDevice = new LinkedHashMap<>();
+      List<String> measurements = new ArrayList<>();
+      Map<String, Set<String>> measurementsGroupByDevice = new LinkedHashMap<>();
       Map<String, TSDataType> dataTypeConsistencyChecker = new HashMap<>();
```

Comment:
```suggestion
      Map<String, TSDataType> measurementToTypeMap = new HashMap<>();
```

Reply:
It's the name in QueryPlan and is used here. Keep it. I will comment it.